### ISSUE TITLE
Change standard sync interval or ask for it on creation

### ISSUE THREAD

**Issue by cdwi** [aegif/CmisSync#536]
As you noted, the 5 seconds sync interval is good for debugging. Unfortunatly many users forget to change that value, causing a high load on the server and the (sometimes mobile) internet connection and filling the Alfresco audit log with a lot of noise - although the use case does not require that short interval.
(At least for us) It would be great to have an higher interval like 30 mins as default - or maybe ask for that parameter in the creation wizard.

**Comment by nicolas-raoul** [comment]
Hello @cdwi 
Thanks for the feedback!

Are you using the latest CmisSync with [ChangeLog activated on Alfresco](https://github.com/aegif/CmisSync/wiki/How-to-enable-CMIS%27-ChangeLog-feature-on-Alfresco)?
The recent ChangeLog improvements have made server load much lighter.

Also, we are working on a management server where CmisSync will connect to get its initial parameters, for instance the sync interval. It will be a good solution for enterprise environments where manually configuring all clients is too time-consuming and unreliable.

Cheers!
Nicolas Raoul

**Comment by cdwi** [comment]
Yes, we have (we talked about it in Google Groups - sorry forgot to set my real name here :-).
But it's still preassure on the system (also remember the "delete" problem in Alfresco), the audit log - and the wire: many users have only access through UMTS with a limited quota.
Such a central server is a good idea. But I thought that adding that parameter to the -already existing- wizard might also be a quick solution.
Another idea might be to store configuration parameters somewhere in the repository, so no dedicated server would be required...
Just my 2 cent
Daniel Winkler

**Comment by nicolas-raoul** [comment]
The wizard is designed to be ultra-simple. That's why we even try to recognize the server instead of asking for the exact CMIS URL. Users can change the poll interval in the settings afterwards.

Initially we have thought about storing configuration on the repository itself, but did not find a solution that would be accepted by everyone, unfortunately.
